src/uct/rocm/copy/rocm_copy_cache.c

```diff
@@ -0,0 +1,228 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2018. ALL RIGHTS RESERVED.
```

Comment:
does it need to be 2022 as well, because it is a new file?

Reply:
done